lib/destination/ddl/expiry.go

```diff
@@ -5,21 +5,31 @@ import (
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/artie-labs/transfer/lib/config/constants"
 )
 
 func ShouldDeleteFromName(name string) bool {
-	nameParts := strings.Split(name, "_")
-	if len(nameParts) < 2 {
+	parts := strings.Split(strings.ToLower(name), constants.ArtiePrefix)
+	if len(parts) != 2 {
+		return false
+	}
+
+	suffixParts := strings.Split(parts[1], "_")
+	if len(suffixParts) != 3 {
 		return false
 	}
 
-	unixString := nameParts[len(nameParts)-1]
-	unix, err := strconv.Atoi(unixString)
+	tsString := suffixParts[len(suffixParts)-1]
```

Comment:
Could just hardcode this to `[2]` if we're enforcing that `len(suffixParts)` has to be exactly 3, but I think it's also fine as is.

Reply:
True